lib/engineyard/cli.rb

```diff
@@ -522,6 +522,47 @@ def ssh(cmd=nil)
       exit exits.detect {|status| status != 0 } || 0
     end
 
+    desc "console [--environment ENVIRONMENT", "Open a Rails console session to the master app server."
+    long_desc <<-DESC
+      Opens a Rails console session on app master.
+    DESC
+    method_option :environment, type: :string, aliases: %w(-e),
+      required: true, default: '',
+      desc: "Environment to console into"
+    method_option :account, type: :string, aliases: %w(-c),
+      required: true, default: '',
+      desc: "Name of the account in which the environment can be found"
```

Comment:
Need to add the `method_option :app` also. You're using `fetch_app_environment` which is the correct fetcher, but it works better if it has the application name as an option (since, in particular, you want to open a specific application's rails console).